prototype2/common/EV42Serializer.h

```diff
@@ -0,0 +1,72 @@
+/// Copyright (C) 2016-2018 European Spallation Source, see LICENSE file
+//===----------------------------------------------------------------------===//
+///
+/// \file
+///
+/// \brief flatbuffer serialization
+///
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "ev42_events_generated.h"
+#include "Producer.h"
+#include <common/Buffer.h>
+#include <functional>
```

Comment:
I dont see the need for this include here - maybe move to cpp file if this where it 
is used?